controllers/projects_controller.go

```diff
@@ -217,8 +189,11 @@ func (u *Projects) FetchGithub(c web.C, w http.ResponseWriter, r *http.Request)
 		for _, l := range lists {
 			jsonLists = append(jsonLists, &ListJsonFormat{Id: l.Id, ProjectId: l.ProjectId, UserId: l.UserId, Title: l.Title.String, ListTasks: TaskFormatToJson(l.Tasks()), Color: l.Color.String, ListOptionId: l.ListOptionId.Int64})
 		}
+		noneList := project.NoneList()
+		jsonNoneList := &ListJsonFormat{Id: noneList.Id, ProjectId: noneList.ProjectId, UserId: noneList.UserId, Title: noneList.Title.String, ListTasks: TaskFormatToJson(noneList.Tasks()), Color: noneList.Color.String, ListOptionId: noneList.ListOptionId.Int64}
```

Comment:
__[gometalinter]__
* [structcheck] unused struct field cannot pass argument noneList.Tasks() (value of type []*../task.TaskStruct) to parameter of type []*../models/task.TaskStruct
* [aligncheck] cannot pass argument noneList.Tasks() (value of type []*../task.TaskStruct) to parameter of type []*../models/task.TaskStruct